common/define-grammar.js

```diff
@@ -445,6 +445,7 @@ module.exports = function defineGrammar(dialect) {
         'protected'
       )),
 
+      minus_readonly: $ => '-readonly',
```

Comment:
Based on experiments in the TypeScript playground, I think the `-` (minus sign) is a separate token from the `readonly` keyword. There can be whitespace in between. Can you restructure this so that it reuses the existing `readonly` rule, but with an optional `-` sign?

Maybe something like:

```js
      index_signature: $ => seq(
        optional(seq(
          optional('-'),
          $.readonly
        )),
        // ...
      )
```

Reply:
The suggestion was implemented